templates/typescript/samples/sample-skill/src/index.ts

```diff
@@ -64,21 +66,34 @@ function getTelemetryClient(settings: Partial<IBotSettings>): BotTelemetryClient
     return new NullTelemetryClient();
 }
 
+// Configure configuration provider
+const credentialProvider: SimpleCredentialProvider = new SimpleCredentialProvider(appsettings.microsoftAppId, appsettings.microsoftAppPassword);
+
+// Register AuthConfiguration to enable custom claim validation.
+const allowedCallersClaimsValidator: AllowedCallersClaimsValidator = new AllowedCallersClaimsValidator("...");
```

Comment:
```suggestion
const allowedCallersClaimsValidator: AllowedCallersClaimsValidator = new AllowedCallersClaimsValidator(appsettings.allowedCallers);
```

I look the constructor of [AllowedCallersClaimsValidator](https://github.com/microsoft/botframework-solutions/blob/v1.0/samples/csharp/skill/SkillSample/Authentication/AllowedCallersClaimsValidator.cs) and looks for the `allowedCallers` as [configKey](https://github.com/microsoft/botframework-solutions/blob/v1.0/samples/csharp/skill/SkillSample/Authentication/AllowedCallersClaimsValidator.cs#L19)